apps/main/src/dex/components/PageCreatePool/TokensInPool/SetOracle.tsx

```diff
@@ -26,41 +27,24 @@ type OracleInputProps = {
 }
 
 const SetOracle = () => {
-  const tokenA = useStore((state) => state.createPool.tokensInPool.tokenA)
-  const tokenB = useStore((state) => state.createPool.tokensInPool.tokenB)
-  const tokenC = useStore((state) => state.createPool.tokensInPool.tokenC)
-  const tokenD = useStore((state) => state.createPool.tokensInPool.tokenD)
-  const tokenE = useStore((state) => state.createPool.tokensInPool.tokenE)
-  const tokenF = useStore((state) => state.createPool.tokensInPool.tokenF)
-  const tokenG = useStore((state) => state.createPool.tokensInPool.tokenG)
-  const tokenH = useStore((state) => state.createPool.tokensInPool.tokenH)
+  const tokens = useStore((state) => state.createPool.tokensInPool)
+
+  const oracleTokens: { token: TokenState; tokenId: TokenId; title: string }[] = [
+    { token: tokens.tokenA, tokenId: TOKEN_A as TokenId, title: t`Token A` },
```

Comment:
crazy data model, who thought it was better to have 8 properties instead of an array :sweat: